plugins/chat/app/services/chat/create_message.rb

```diff
@@ -54,6 +55,7 @@ class Contract
       attribute :upload_ids, :array
       attribute :thread_id, :string
       attribute :incoming_chat_webhook
+      attribute :process_inline, :boolean, default: Rails.env.test?
```

Comment:
This can be seen as a hack, but after a long time playing with `Jobs.run_immediately!` and other inline solutions, I decided to go with this. And even if it means prod can be different that tests,  it's already how it is in tests given we use `Jobs.run_immediately!` which won't be the case in prod, having this explicitly in the service has two values:

- you can test the two cases
- you don't have to use `Jobs.run_immediately!` in tests

note that I couldn't remove all the `Jobs.run_immediately!` due to other jobs.